src/gestures.c

```diff
@@ -351,6 +351,10 @@ static void buttons_update(struct Gestures* gs,
 				else if (touching == 3 && cfg->button_3touch > 0)
 					trigger_button_emulation(gs, cfg->button_3touch - 1);
 			}
+			else if (touching == 0 && cfg->button_0touch > 0) {
+				LOG_EMULATED("buttons_update: integrated clicked without touch detection%d\n", touching);
```

Comment:
Although log message is very helpful, in a future it would be useful to have one or two sentences with reason behind that code branch.

Reply:
By this, do you mean that I should add a comment explaining the `if block` inside the code?

Reply:
Yes, please, few words explaining reason, and/or reference to new Readme section or this PR.

Reply:
Sure, and please don't merge yet, I spotted an error: `touching == 0` is always True. I'll fix it this weekend.